.github/workflows/ci.yml

```
@@ -12,166 +12,149 @@ on:
-        run: pnpm install --frozen-lockfile
-      - name: Typecheck
-        run: |
-          if [ "${{ github.event_name }}" = "pull_request" ]; then
-            pnpm exec turbo run typecheck --affected
-          else
-            pnpm run typecheck
-          fi
+          fetch-depth: 0 # --affected needs history to diff against the base
+          persist-credentials: false
+
+      - parallel:
```

actionlint (1.7.12)

[error] step must run script with "run" section or run action with "uses" section

(syntax-check)